### ISSUE TITLE
Add survey-based testing signals to the map

### ISSUE THREAD

**Issue by capnrefsmmat** [cmu-delphi/www-covidcast#559]
Now that the symptom surveys ask respondents if they have been tested and what the results were, we should consider adding these signals to the map. There are three of interest:

* Percentage of people who were tested in the last 14 days (testing volume)
* Percentage of those tests that were positive (test positivity)
* Of people who were *not* tested, percentage who wanted to be tested

I'm not sure which of these we should present and how we should present them. For example, if we show test positivity, how do we clearly distinguish that signal from the Quidel signal?

This probably requires some careful thinking about what signals are most useful for public health. @RoniRos, any opinions here?

**Comment by RoniRos** [comment]
I think that's a feature, not a bug.  Antigen testing by one manufacturer is very different than overall testing.  But even if it weren't, we are in the business of providing different measurements, not reconciling them.  So I think we just need to be very descriptive.  E.g. "COVID Antigen Test Positivity (Quidel)" and "COVID Test Positivity (Survey)".

**Comment by capnrefsmmat** [comment]
I agree it's a feature. But from a user's perspective: When would I want to know specifically antigen test positivity, vs. when would I want to know overall positivity from the survey? Does comparing the two side-by-side help me draw any useful conclusions?

I know we've heard from officials that they already have too much data, so I'm wary of just putting more signals on the map without thinking of how we expect users to use them to make decisions, or if they'd be useful for that decision-making.

**Comment by RoniRos** [comment]
Thee are fair questions.  I guess the answers may differ across different types of users.
Data savvy users may want to know whether the two measurements agree, as a way of assessing their reliability/robustness.
More casual users may prefer that we combine multiple measurements of the same underlying quantities and present only the combined signal.
We are going to face this exact problem also with Doctor Visits (Outpatient) and Hospital Admissions (Inpatient), as soon as our Change HealthCare (CHC) signals are ready.

**Comment by sgratzl** [comment]
is this issue still valid?

**Comment by capnrefsmmat** [comment]
If we add one, it should be `fb-survey` `smoothed_tested_positive_14d`. However, we have not established whether it is comparable to other sources of test positivity rate, or whether it's subject to the same biases some of our other survey signals seem to have. We'd probably want to do that first to better understand it.
